tfhe/src/integer/gpu/mod.rs

```diff
@@ -480,9 +495,20 @@ pub fn get_scalar_mul_integer_radix_kb_size_on_gpu<T: UnsignedInteger>(
     num_blocks: u32,
     pbs_type: PBSType,
     grouping_factor: LweBskGroupingFactor,
-    noise_reduction_key: Option<&CudaModulusSwitchNoiseReductionKey>,
+    ms_configuration: Option<&CudaModulusSwitchNoiseReductionConfiguration>,
```

Comment:
There are quite many I let you make the change for all if the name I'm suggesting is ok for you :slightly_smiling_face: